internal/conf/config.yaml

```
@@ -60,8 +60,11 @@ realtime:
 
   dashboard:
     thumbnails:
-      summary: false
-      recent: true
+      debug: false        # true to enable debug mode for image provider
+      summary: false      # show thumbnails on summary table
+      recent: true        # show thumbnails on recent table
+      imageprovider: auto # preferred image provider: auto, wikimedia, avicommons
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)